src/practices/Go/GoGitignoreCorrectlySetPractice.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { GoGitignoreCorrectlySetPractice } from './GoGitignoreCorrectlySetPractice';
+import { gitignoreContent } from '../../detectors/__MOCKS__/Go/gitignoreContent.mock';
+import { PracticeEvaluationResult } from '../../model';
+import {
+  TestContainerContext,
+  createTestContainer,
+} from '../../inversify.config';
+
+const basicGitignore = `
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+*.test
+*.out
+vendor/
+`;
+
+describe('GoGitignoreCorrectlySetPractice', () => {
+  let practice: GoGitignoreCorrectlySetPractice;
+  let containerCtx: TestContainerContext;
+
+  beforeAll(() => {
+    containerCtx = createTestContainer();
+    containerCtx.container
+      .bind('GoGitignoreCorrectlySetPractice')
+      .to(GoGitignoreCorrectlySetPractice);
+    practice = containerCtx.container.get('GoGitignoreCorrectlySetPractice');
+  });
+
+  afterEach(async () => {
+    containerCtx.virtualFileSystemService.clearFileSystem();
+    containerCtx.practiceContext.fileInspector!.purgeCache();
+  });
+
+  it('Returns practicing if the .gitignore is set correctly', async () => {
+    containerCtx.virtualFileSystemService.setFileSystem({
+      '.gitignore': gitignoreContent,
+    });
+
+    const evaluated = await practice.evaluate(containerCtx.practiceContext);
+    expect(evaluated).toEqual(PracticeEvaluationResult.practicing);
+  });
+
+  it('Returns notPracticing if there the .gitignore is NOT set correctly', async () => {
```

Comment:
Returns notPracticing if the .gitignore is NOT set correctly

Reply:
Made the requested change in the latest push